deploy/gk-deploy

```diff
@@ -214,24 +234,26 @@ if [ "x${CLI}" == "x" ]; then
   fi
 fi
 
-if [ "${CLI}" == "oc" ]; then
+CLI="${CLI} -n ${namespace}"
+
+if [ "${CLI}" == *oc* ]; then
   output "Starting OpenShift deployment."
-  oc new-project heketi
+  ${CLI} new-project heketi
   oadm policy add-scc-to-user privileged -z default
-  oc create -f /usr/share/heketi/templates
+  ${CLI} create -f /usr/share/heketi/templates
```

Comment:
Please open a new Issue, I'll take care of it when I actually do the openshift integration properly. :)